src/Nethermind/Nethermind.Blockchain.Test/Synchronization/BlockDownloaderTests.cs

```diff
@@ -21,6 +21,7 @@
 using System.Linq;
```

Comment:
can we also add tests that verify that receipts are downloaded? for now I can only see the update of existing tests to make them work